shared/protocol/src/test/java/io/pravega/shared/NameUtilsTest.java

```diff
@@ -73,6 +74,23 @@ public void testStreamNameVerifier() {
         NameUtils.validateStreamName("1.2.3");
     }
 
+    @Test
+    public void testStreamNameLimit() {
+        int leftLimit = 48; // numeral '0'
+        int rightLimit = 122; // letter 'z'
+        int targetStringLength = 256;
+        Random random = new Random();
```

Comment:
seed with 0

Reply:
Aside: It might be cool to bring in the *jqwik* test engine so we can get property generators such as string length and character classes: https://jqwik.net/docs/current/user-guide.html#string-length

Cf., https://github.com/junit-team/junit5-samples/tree/main/junit5-multiple-engines

Reply:
Since the test is written around a hard coded limit, I think bringing in jqwik for this might be a bit of an overkill.